editions/tw5.com/tiddlers/Operators without parameters.tid

```diff
@@ -0,0 +1,26 @@
+created: 20240708171243370
+modified: 20240708185727902
+tags: 
+title: Operators without parameters
+
+Many [[Filter Operators]] have no [[parameter|Filter Parameter]] available. Still, each operator must be followed by a bracketed parameter expression — even if it is empty — as with the <<.olink backlinks>> operator below:
+
+`[<currentTiddler>backlinks[]]`
+
+(Even though an expression such as `[<currentTiddler>backlinks]` may at first //seem// well-formed — insofar as closing brackets seem to pair properly with opening brackets — each operator needs its own parameter brackets, even if empty. See [[Filter Syntax]].)
+
+The following [[Filter Operators]] accept no parameters:
+
+<div class="sh-multicol">
+
+<<list-links filter:"[op-parameter[none]] [tag[Filter Operators]!has[op-parameter]] -[search:op-purpose[same]]">>
+
+</div>
+
+<style>
+.sh-multicol {
+  column-width: 14em;
+  column-rule: 1px solid #ccc;
+}
+.sh-multicol  > :first-child { margin-top: 0;}
+</style>
```

Comment:
We do have an existing class. See: 
![image](https://github.com/Jermolene/TiddlyWiki5/assets/374655/eef8556b-4617-4280-a35b-dc86076e0c15)

Reply:
My bad. I've updated the PR with multi-columns